### PR TITLE
Release 2.1.8

### PR DESCRIPTION
**Pull request by jakirkham**
```
2017-03-24 2.1.8:

Enhancements:
-------------

* use symlinks when copying files from files sources to improve performance  #1867

Bug fixes:
----------

* reset build folder for each built package (fixes building multiple recipes in one command)  #1842
* wrap copy of test/source_files so that errors don't prevent a successful build  #1843
* Restore permissions when extracting from zipfiles  #1855
* pass through several Windows-specific environment variables  #1859, #1862
* python 2 os.environ string type compatibility fix  #1861
* fix indentation breaking perl package testing  #1872
* reduce scope of git try/except block so that GIT_FULL_HASH is available even if tags are not.  #1873
* fix license family comparison, especially for public-domain  #1875
* Remove python header being added to all run_test.* files  #1876

Contributors:
-------------

* @abretaud
* @jjhelmus
* @mingwandroid
* @msarahan
* @stuertz
* @wulmer
```

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ requirements:
   run:
     - beautifulsoup4
     - chardet
-    - conda  >=4.1
+    - conda  >=4.1,<4.4
```
Comment (jakirkham) [inline recipe/meta.yaml]: Accidentally missed this in PR ( https://github.com/conda-forge/conda-build-feedstock/pull/20 ). Am adding it now.

**Comment by jakirkham** [comment]
Does this look ok to you, @msarahan?

**Comment by jakirkham** [comment]
Friendly nudge @msarahan. 😉

**Comment by jakirkham** [comment]
Just going to merge. Hope that is ok.

**Comment by msarahan** [comment]
Yes, this was fine. Sorry, was it on vacation last week.

**Comment by jakirkham** [comment]
No worries. Thanks for getting back to me. Hope you had a good vacation.

**Pull request merged**